src/Symfony/Bundle/FrameworkBundle/Tests/Controller/ControllerTest.php

```diff
@@ -98,7 +98,6 @@ public function testGetUserWithEmptyContainer()
     }
 
     /**
-     * @param $token
      *
      * @return ContainerInterface
      */
```

Comment:
I would drop the whole, eventually add a `TokenInterface` typehint to `$token` and a return type.